fe/src/main/java/org/apache/doris/analysis/MVColumnItem.java

```diff
@@ -79,4 +94,19 @@ public Expr getDefineExpr() {
     public void setDefineExpr(Expr defineExpr) {
         this.defineExpr = defineExpr;
     }
+
+    public Column toMVColumn(OlapTable olapTable) throws DdlException {
+        Column baseColumn = olapTable.getColumn(name);
+        if (baseColumn == null) {
+            Preconditions.checkNotNull(defineExpr != null);
+            Column result = new Column(name, type, isKey, aggregationType, ColumnDef.DefaultValue.ZERO, "");
+            result.setDefineExpr(defineExpr);
+            return result;
+        } else {
```

Comment:
baseColumn may be is not null , but we should set define expr in column. such as two different materialized view aggregate function use same base filed .